CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Updated to CircleCI 2.1 configuration syntax (#3512)
 - Adjusted how we present the BonApp ultimatum on first visiting the Balances tab (#3515)
 - Changed the `data` prop on fancy-menu to be `extraData` (#3528)
+- Upgraded to RN 0.59 (#3557)
```

Comment:
Does this need to be abbreviated?

Reply:
It's the developer changelog, for us; I don't particularly care, but I will forget and abbreviate it next time as well.